src/core/Notification/index.tsx

```diff
@@ -0,0 +1,47 @@
+import { AlertProps } from "@material-ui/lab";
+import React from "react";
+import { ReactComponent as IconAlert } from "../../common/svgs/IconAlert.svg";
+import { ReactComponent as IconClose } from "../../common/svgs/IconClose.svg";
+import { ReactComponent as IconSuccess } from "../../common/svgs/IconSuccess.svg";
+import { StyledButton, StyledNotification } from "./style";
+
+export { AlertProps };
+
+export interface NotificationProps {
+  sdsAction?: (event: React.SyntheticEvent) => void;
+  sdsActionText?: string;
```

Comment:
maybe `text` would do?

Reply:
I think just `text` might be a bit vague in the scope of the whole component. What about `buttonText`?

Reply:
Oh yes that's even better 🚀 ! Great idea thank you, Meagan 😄 !